### DISCUSSION TITLE
Trying to setup Quasar for the First time

### DISCUSSION THREAD

**Discussion by k24h1** [quasarframework/quasar#11853] [Quasar App Extensions]
Hi there, I'm interested in setting Quasar and I tried watching a hackersploit video from 2017, the files on github have changed since then and there is no direct installation via a batch file like there was back then, could some one please tell me how I can download and set this up now? if you have the version of the files from back then or know what to do, please let me know

**Comment by smolinari** [comment]
@k24h1 - So, you are simply trying to get a Quasar project going? 

Scott

**Comment by hawkeye64** [comment]
Consider checking out this video course (it's free!)
https://quasarcast.com/course
